strikt-core/src/test/kotlin/strikt/assertions/IterableAssertions.kt

```diff
@@ -217,6 +217,7 @@ internal class IterableAssertions {
         }
       }
 
+    @TestFactory
```

Comment:
D’oh! Did I miss that?

Reply:
i found it in the UnusedSymbols report of Ideas analyze code output.